dataprofiler/profilers/profile_builder.py

```diff
@@ -1112,6 +1114,19 @@ def _save_helper(self, filepath: str | None, data_dict: dict) -> None:
         # Restore all data labelers
         self._restore_data_labelers(data_labelers)
 
+    def _json_save_helper(self, filepath: str) -> None:
+        """
+        Save profiler to disk.
+
+        :param filepath: Path of file to save to
+        :type filepath: String
+        :return: None
+        """
+        if filepath is None:
+            filepath = ""
+        with open(filepath, "w") as f:
```

Comment:
don't we need some kind of file name at least for this because `with open("", "w") as f:`, even if it did succeded wouldn't be ideal for finding the thing it wrote

Reply:
see suggestion above

Reply:
this is sneaky because it probably silently runs but doesn't do what we actually want ... 💀

Reply:
good callout